collectors/ebpf.plugin/ebpf_swap.c

```diff
@@ -296,6 +331,202 @@ void ebpf_swap_send_apps_data(struct target *root)
     write_end_chart();
 }
 
+/**
+ * Sum PIDs
+ *
+ * Sum values for all targets.
+ *
+ * @param swap
+ * @param root
+ */
+static void ebpf_swap_sum_cgroup_pids(netdata_publish_swap_t *swap, struct pid_on_target2 *pids)
+{
+    uint64_t local_read = 0;
+    uint64_t local_write = 0;
+
+    while (pids) {
+        netdata_publish_swap_t *w = &pids->swap;
+        local_write += w->write;
+        local_read += w->read;
+
+        pids = pids->next;
+    }
+
+    // These conditions were added, because we are using incremental algorithm
+    swap->write = (local_write >= swap->write) ? local_write : swap->write;
+    swap->read = (local_read >= swap->read) ? local_read : swap->read;
```

Comment:
I think the logic in this function suffers from the same issue we described in the shared-memory eBPF PR, right? It only counts currently existing counts but not for PIDs that have been deleted, which we need to do for the incremental algorithm.

Reply:
Yes, you are right.
I won't address this issue with this PR, I will prepare one exclusively to change all threads that has this logic.

Reply:
Alright!